nextflow.config

```diff
@@ -51,7 +51,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,fasta'
+    validationSchemaIgnoreParams     = 'genomes,igenomes_base,fasta'
```

Comment:
@jfy133 Do we need to keep both `genome` and `igenomes_base`? The template includes both of them. We also appended the `fasta`.

Reply:
Yes, we currently don't use either (although maybe should at some point)... so keep them under ignore